src/animators/animator.ts

```diff
@@ -1,5 +1,39 @@
 module Plottable {
+export module Animators {
```

Comment:
I'm more thinking of adding these types after the `Animator` interface so that essentially we have

```typescript
module Plottable {
  export interface Animator {
  }
}

module Plottable.Animators {
  export stuff {
  }
}
```

I am more inclined in this way since I feel that it exposes namespaces a lot better.